### DISCUSSION TITLE
Livewire wire:click destroy owl carousel

### DISCUSSION THREAD

**Discussion by Hacker-Nahid-360** [livewire/livewire#3421] [1. Help]
Hello,
I am having a problem with livewire and owl carousel. owl carousel loading perfectly and works fine with the livewire component. I set a wire:click and when anyone click on the button owl carousel is destroyed. It not showing properly.

**Comment by joshhanley** [comment]
@Hacker-Nahid-360 any JS library that modifies the DOM will get wiped out anytime Livewire does a request. 

You will need to make sure of `wire:ignore` to stop that happening.

Have a good read of everything on this page as it explains what happens and how to use `wire:ignore` https://laravel-livewire.com/docs/2.x/alpine-js
If you have access to the screencasts, check out this one https://laravel-livewire.com/screencasts/s6-refactor-pikaday-trix

Hope this helps!
